src/Catalyst.Core.Modules.KeySigner.Tests/UnitTests/KeySignerTests.cs

```diff
@@ -148,6 +149,7 @@ private sealed class CryptoContext : ICryptoContext
 
             public ISignature Sign(IPrivateKey privateKey, ReadOnlySpan<byte> message, ReadOnlySpan<byte> context) => _signature;
             public bool Verify(ISignature signature, ReadOnlySpan<byte> message, ReadOnlySpan<byte> context) { throw new NotImplementedException(); }
+            public bool BatchVerify(IList<ISignature> signatures, IList<byte[]> messages, ReadOnlySpan<byte> context) { throw new NotImplementedException(); }
```

Comment:
are test ignored?

Reply:
Yeah we don't need an implementation of batch verification in the KeySigner tests